ui/components/app/assets/asset-list/asset-list.tsx

```diff
@@ -115,7 +118,13 @@ const AssetList = ({ onClickAsset, showTokensLinks }: AssetListProps) => {
         )}
       <AssetListControlBar showTokensLinks={showTokensLinks} />
       <TokenList
-        nativeToken={<NativeToken onClickAsset={onClickAsset} />}
+        nativeToken={
+          <>
+            {chains?.map((chain) => (
+              <NativeToken chain={chain} onClickAsset={onClickAsset} />
+            ))}
+          </>
+        }
```

Comment:
Let's rename this prop to `nativeTokens`

Rather than wrapping them in a fragment, let's just pass them in as an array of `NativeToken` elements
```suggestion
        nativeToken={chains?.map((chain) => (
          <NativeToken chain={chain} onClickAsset={onClickAsset} />
        ))}
```

This will allow us to render each of them independently farther down the component tree, since they won't be bundled together in a fragment. They will need to appear in different orders in the list depending on sort and filter criteria

Reply:
Also, if you see an alternate approach to solve for this problem, I'm open to discussing that as well 👍